### ISSUE TITLE
not a bug: more a feedback with windows machine to be used by others

### ISSUE THREAD

**Issue by alain2208** [drscotthawley/panotti#31]
Hi i discovered this git few days ago, seems very nice job!

I work on windows and not a great pyhton guy but very interested in voice.

For now the main issues i found are related to path. Python windows don't like a lot relative path and so for now i try to 'make it work' in rewriting path in full not in relative.
Also have to add in the environment variable the path to ffmpeg that i have to grab for librosa to work with mp3.

**Comment by drscotthawley** [comment]
@alain2208 Ok, cool!  Send me a PR if you get it working. 
Preferably the new code could be wrapped in code that tests for the OS. ;-)

**Comment by f00f** [comment]
I got panotti running on Windows without changing relative paths (but maybe that has been merged in the meantime).
I had python 3.6.8, Keras 2.3.1and tensorflow-gpu 2.0.0 installed an running already.
Note that I only got panotti running and have not evaluated its performance, yet.

- clone the [panotti](https://github.com/drscotthawley/panotti) repo
- install `sox`, add to `PATH`
- download `ffmpeg`, add to `PATH`

Make sure to open new PowerShell and git bash windows after changing `PATH`.

Edit `./train_network.py` and `./preprocess_data.py`: change first lines to
```python
#! /usr/bin/env python
```
(i.e. `python3` -> `python`)

In PowerShell:
```powershell
pip install librosa==0.6.3
pip install cython matplotlib imageio pandas audioread h5py librosa Pillow pypandoc scikit_learn winshell scikit-image
```
The recent libRosa crashed somewhere in `binaural_setup.sh`.

In git bash:

```bash
cd examples
cd ./binaural_setup.sh
```

Tensorflow complains about dropout values > 0.5:
> WARNING:tensorflow:Large dropout rate: 0.6 (>0.5). In TensorFlow 2.x, dropout() uses dropout rate instead of keep_prob. Please ensure that this is intended.

so I guess this needs to be changed to 0.4.

Hope this helps someone.

**Comment by drscotthawley** [comment]
Thanks for sharing this!  I'll leave this up for others...

**Comment by drscotthawley** [comment]
Since this is solved, I'll close. They can still find it in searching closed issues.
